app/assets/javascripts/proxy_status.js

```diff
@@ -95,3 +103,29 @@ function populateData(response, item) {
     setItemStatus($(this), response);
   });
 }
+
+function setTab(){
+  var anchor = document.location.hash;
+  if (anchor.length) {
+    var parent_tab = $(anchor).parents('.tab-pane');
+    if (parent_tab.exists()){
+      $('.nav-tabs a[href=#'+parent_tab[0].id+']').tab('show');
+    }
+    $('.nav-tabs a[href='+anchor+']').tab('show');
+  }
+}
+
+function refreshCharts(){
+  $(".statistics-bar").flot_bar();
+  $(".statistics-pie").flot_pie();
+}
+
+function filterCerts(state) {
+  var table = $('#certificates table').dataTable();
+  table.fnFilter(state, 1);
+}
+
+function certTable() {
+  activateDatatables();
+  filterCerts('valid');
```

Comment:
or in other words, can you add a counter for how many pending certs are there, and if you click on it redirects you to the list ? :)